catalogue/webapp/components/WorkDetailsLinks/WorkDetailsLinks.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import NextLink from 'next/link';
+import Space from '@weco/common/views/components/styled/Space';
+import { classNames } from '@weco/common/utils/classnames';
+import WorkDetailsProperty from '../WorkDetailsProperty/WorkDetailsProperty';
+
+// TODO: Fix links type
```

Comment:
do we not have a `Link` type already, with `url` and `text` keys?

Reply:
Copy / paste